hs_file_types/models/reftimeseries.py

```diff
@@ -789,6 +789,7 @@ def set_file_type(cls, resource, user, file_id=None, folder_path=None):
         try:
             json_file_content = _validate_json_file(res_file)
         except Exception as ex:
+            log.exception("failed json validation")
```

Comment:
@sblack-usu Since ValidataionError() is raised, I think this log statement can be removed - I know logging is a good thing, but too much logging makes finding useful info more difficult, so I think we should only log useful information if possible on production

Reply:
I think this is a good thing to log.  This only occurs when the json validation fails in a .refts.json file.  It's nice to be able to lookup where it failed in the logs.

We lose the stacktrace when an error is raised with the exception.message.

Reply:
OK. Agreed it is a good thing to be able to see stack trace in the log.